src/main/java/core/basesyntax/Trapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Trapezoid extends Figure implements Drawable {
+
+    private double baseA;
```

Comment:
why in some classes your figure parameters are final and in some not. Let's keep code consistent and use same approach everywhere unless there is a particular reason not to do it